module/users/validator/v1.js

```diff
@@ -3,6 +3,9 @@ module.exports = (req) => {
 		solutions: function () {
 			req.checkParams('_id').exists().withMessage('required Program id')
 		},
+		programs: function () {
```

Comment:
@Prajwal17Tunerlabs we can remove this if not required

Reply:
removed unwanted validations.